seqr/views/utils/terra_api_utils.py

```diff
@@ -77,7 +67,14 @@ def remove_token(user):
 
 
 def is_anvil_authenticated(user):
-    return anvil_enabled() and is_google_authenticated(user)
+    if not anvil_enabled():
+        return False
+
+    social = _safe_get_social(user)
+    if social and social.extra_data:
+        return social.extra_data.get('access_token', '') != ''
```

Comment:
I've just recognized that it would be even better to add a token expiration test like:`social.extra_data['auth_time'] + social.extra_data['expires'] - 10) <= int(time.time())`. And, we could define a helper function for the test and re-use it with `_get_social_access_token()`.

Reply:
this is out of scope of this PR. I am happy to have a conversation offline about why I think that is a bad idea